packages/bot/src/stores/google-drive-store.js

```diff
@@ -0,0 +1,213 @@
+import { importExternal }   from '../utils/tools';
+
+export default class GoogleDriveStore {
+    constructor(root_store) {
+        this.root_store = root_store;
+        this.bot_folder_name = `Binary Bot - ${translate('Strategies')}`;
+        this.google_auth = null;
+        this.setKey();
+        
+        importExternal('https://apis.google.com/js/api.js')
+            .then(() => this.initialise());
+    }
+
+    @observable is_authorised = false;
+
+    setKey = () => {
+        const { aid, cid, api } = config.gd;
+        this.client_id = cid;
+        this.app_id = aid;
+        this.api_key = api;
+    }
+
+    initialise() {
+        gapi.load('client:auth2:picker', {
+            callback: () => {
+                gapi.client
+                    .init({
+                        apiKey       : this.api_key,
+                        clientId     : this.client_id,
+                        scope        : 'https://www.googleapis.com/auth/drive',
+                        discoveryDocs: ['https://www.googleapis.com/discovery/v1/apis/drive/v3/rest'],
+                    })
+                    .then(
+                        () => {
+                            this.google_auth = gapi.auth2.getAuthInstance();
+                            this.google_auth.isSignedIn.listen(is_signed_in => this.updateSigninStatus(is_signed_in));
+                            this.updateSigninStatus(this.google_auth.isSignedIn.get());
+                        },
+                        error => {
+                            // TODO
+                            console.error(error); // eslint-disable-line
+                        }
+                    );
+            },
+            onerror: console.error, // eslint-disable-line
+        });
+    }
+
+    @action.bound
+    updateSigninStatus(is_signed_in) {
+        this.is_authorised = is_signed_in;
+    }
+    
+    signIn() {
+        if (this.is_authorised) {
+            return Promise.resolve();
+        }
+
+        return this.google_auth.signIn({ prompt: 'select_account' });
+    }
+
+    signOut() {
+        if (!this.is_authorised) {
+            return Promise.resolve();
+        }
+
+        return this.google_auth.signOut();
+    }
+
+    // eslint-disable-next-line class-methods-use-this
+    getPickerLanguage() {
+        const language = getLanguage();
+
+        if (language === 'zhTw') {
+            return 'zh-TW';
+        } else if (language === 'zhCn') {
+            return 'zh-CN';
+        }
+        return language;
+    }
+
+    @action.bound
+    async saveFile(options) {
+        await this.signIn();
+        await this.checkFolderExists();
+
+        await this.createSaveFilePicker('application/vnd.google-apps.folder', 'Select a folder', options);
+    }
+
+    @action.bound
+    async loadFile() {
+        await this.signIn();
+        const xml_doc = await this.createLoadFilePicker(['text/xml', 'application/xml'], 'Select a Deriv Bot Strategy');
```

Comment:
Please use `translate` where necessary.

```suggestion
        const xml_doc = await this.createLoadFilePicker(['text/xml', 'application/xml'], translate('Select a Deriv Bot Strategy'));
```